.github/workflows/e2ev2.yaml

```diff
@@ -0,0 +1,54 @@
+name: E2E Version 2 Test
+
+on:
+  workflow_call:
+    inputs:
+      ref:
+        type: string
+      skipRefCheck:
+        type: boolean
+        default: true
+
+permissions:
+    id-token: write
+
+jobs:
+  plan-infra:
+    runs-on: ubuntu-latest
+    outputs:
+       matrix: ${{ steps.matrix.outputs.matrix }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ inputs.ref }}
+
+      - uses: actions/setup-go@v4
+        with:
+          go-version: '~1.20.3'
+          cache-dependency-path: "**/*.sum"
+
+      - run: |
+          cd testing/e2e
+          go run ./main.go matrix
+        shell: bash
+        id: matrix
```

Comment:
I imagine it's a github glitch that the formatting isn't applying to the keys here and not an indentation issue?

Reply:
looks fine to me and works